CONTRIBUTING.md

```diff
@@ -49,13 +49,8 @@ You can execute the following commands in the following directories:
 - packages/zitadel-next
 - The projects root directory: all tests in the project are executed
 
-```sh
-# Run all once
-pnpm test
-
-# Rerun tests on file changes
-pnpm test:watch
-```
+In apps/login, these commands also spin up the application and a ZITADEL gRPC API mock server to run integration tests using [Cypress](https://www.cypress.io/) against them.
+If you want to run the integration tests standalone against an environment of your choice, navigate to ./apps/login, create your .env.local file and run `pnpm test:integration:run` or `pnpm test:integration:open`.
```

Comment:
shouldnt we add that you have to configure the `.env.local` occording to the mock or provide a default mock.env as the mock is running locally and on 22222 anyways? 🤔

Reply:
I like the idea of a .env.mock file. Will try that.

Reply:
done with https://github.com/zitadel/typescript/pull/28/commits/ba98fde885b214a21f75642ad23fcb075aca0c32
removed encouraging .env.local because it has higher priority than process.env which makes it very complicated to overwrite environments for integration tests and later acceptance tests.

Reply:
nice thanks.. contribution guide is nice like this too 👍